kedro-exercises/spaceflight/src/requirements.txt

```diff
@@ -5,7 +5,7 @@ isort>=4.3.21, <5.0
 jupyter~=1.0
 jupyter_client>=5.1,<7.0
 jupyterlab==0.31.1
-kedro[pandas.CSVDataSet,pandas.ExcelDataSet]==0.16.6
+kedro[pandas.CSVDataSet,pandas.ExcelDataSet]==0.17.0
```

Comment:
Does it use `pandas.ExcelDataSet` dataset?

Reply:
Yup the `shuttles` dataset uses it.